Students/Iuga Cosmin/Lab5/src/pack/Customer.java

```diff
@@ -0,0 +1,62 @@
+package pack;
+
+import java.util.ArrayList;
+
+public class Customer {
+    private int id;
+    private String name;
+    private String contactInfo;
+    private ArrayList<BankAccount> bankAccounts;
+    public Customer(int id,String name,String contactInfo)
+    {
+        this.id=id;
+        this.name=name;
+        this.contactInfo=contactInfo;
+        this.bankAccounts=new ArrayList<>();
+    }
+    public void addBankAccount(BankAccount bankAccount)
+    {
+        bankAccounts.add(bankAccount);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void makeTransaction(int id, BankAccount source, BankAccount destination, String type, double amount)
+    {
+        Transaction transaction = new Transaction(id,source,destination,type,amount);
+        if(type.equals("Deposit"))
+        {
+            destination.addTransaction(transaction);
+            destination.setBalance(destination.getBalance()+amount);
+        }
+        if (type.equals("Withdraw"))
+        {
+            if(source.getBalance()>=amount) {
+                source.addTransaction(transaction);
+                source.setBalance(source.getBalance() - amount);
+            }
+        }
+        if (type.equals("Transfer"))
```

Comment:
e mai safe sa faci comparatia invers `"Transfer".equals(type)`. in cazul in care type e `null`